ports/vcpkg-tool-meson/vcpkg-port-config.cmake

```
@@ -0,0 +1 @@
+set(MESON_SCRIPT "${CMAKE_CURRENT_LIST_DIR}/../../tools/meson/meson.py")

```

```suggestion
set(SCRIPT_MESON "${CMAKE_CURRENT_LIST_DIR}/../../tools/meson/meson.py")
```
Assuming you were trying to interface with `vcpkg_find_acquire_program`?

---

yeah. I originally wanted to set `MESON` but that requires `vcpkg_find_acquire_program(PYTHON3)` which seems to not work in `port-config`s due to include order in `ports.cmake`